### PR TITLE
Balance Battleship

### PR DESCRIPTION
**Pull request by WilliamWeberBerrutti**
**NOTE:** This PR ideally should be merged together with [Balance Mothership](https://github.com/OpenHV/OpenHV/pull/998).

### CODE REVIEW

File: mods/hv/rules/aircraft.yaml [hunk 1]
```diff
@@ -811,12 +811,13 @@ BATTLESHIP:
 	Buildable:
 		Queue: Aircraft
 		BuildPaletteOrder: 100
-		Prerequisites: ~starport.yi, techcenter, ~disabled
+		BuildLimit: 1
+		Prerequisites: radar, trader, techcenter
```
Comment (OcelotWalrus) [inline mods/hv/rules/aircraft.yaml:815]: You made it available to all factions.

Comment (OcelotWalrus) [reply mods/hv/rules/aircraft.yaml:815]: You must also add the `~starport.x` prerequisite or it'll still appear even if the player has no starport (if I'm right)

File: mods/hv/rules/aircraft.yaml [hunk 2]
```diff
@@ -811,17 +811,18 @@ BATTLESHIP:
...
 	Health:
-		HP: 100000
+		HP: 250000
 	Armor:
 		Type: Heavy
 	RevealsShroud:
-		Range: 12c0
-		MinRange: 4c0
+		Range: 16c0
+		MinRange: 6c0
```
Comment (OcelotWalrus) [inline mods/hv/rules/aircraft.yaml:825]:
```suggestion
		MinRange: 8c0
```

From what I've seen, `MinRange` is always the halved value of `Range`.

**Review by OcelotWalrus — CHANGES_REQUESTED** [review]

File: mods/hv/weapons/ballistics.yaml [hunk 3]
```diff
@@ -351,16 +351,11 @@ BunkerMortarGun:
...
+	ReloadDelay: 110
 	Range: 16c0
 	MinRange: 8c0
-	# IDEA: Maybe balance as a single attack, then think on varying its bursts! It should be easier this way!
-	# # Burst: 2
-	# # BurstDelays: 8
 	Burst: 2
 	BurstDelays: 0
-	# Burst: 8
-	# BurstDelays: 8
```
Comment (Mailaender) [inline mods/hv/weapons/ballistics.yaml]: Please merge this into your previous commit.

File: mods/hv/rules/aircraft.yaml (hunk 2 above)
Comment (WilliamWeberBerrutti) [reply mods/hv/rules/aircraft.yaml:825]: It's not always the case, but I think we should change to always be the half of `Range` for all units to be consistant.

**Pull request merged**